### PR TITLE
Add filter types list_of_objects/morphisms/twocells

### PR DESCRIPTION
**Pull request by zickgraf**
It would be good to check the substitutions (`IsList` -> `list_of_objects` or `list_of_morphisms`) which are not covered by the validator a second time. Note that I have excluded some occurrences of `IsList` in the method record (e.g. `HomologyObjectFunctorialWithGivenHomologyObjects`) since I'm not sure if the semantics would be appropriate there.

### CODE REVIEW

File: CAP/gap/MethodRecord.gi [hunk 1]
```diff
@@ -2240,7 +2240,7 @@ AstrictionToCoimageWithGivenCoimage := rec(
 
 UniversalMorphismIntoCoimage := rec(
   installation_name := "UniversalMorphismIntoCoimage",
-  filter_list := [ "morphism", IsList ],
+  filter_list := [ "morphism", "list_of_morphisms" ],
```
Comment (sebastianpos) [inline CAP/gap/MethodRecord.gi]: Depending on your application of this new specification, you may have to be careful here. This list of morphisms is not of arbitrary length, but has to be of length 2 (it stands for a possible factorization of the first argument).

File: CAP/gap/MethodRecord.gi [hunk 2]
```diff
@@ -2545,7 +2545,7 @@ CoastrictionToImageWithGivenImageObject := rec(
 
 UniversalMorphismFromImage := rec(
   installation_name := "UniversalMorphismFromImage",
-  filter_list := [ "morphism", IsList ],
+  filter_list := [ "morphism", "list_of_morphisms" ],
```
Comment (sebastianpos) [inline CAP/gap/MethodRecord.gi]: Same applies here (and of course in the WithGiven cases)

File: CAP/gap/MethodRecord.gi (hunk 1 above)
Comment (zickgraf) [reply CAP/gap/MethodRecord.gi]: You are right, I have reverted the four occurrences. Thanks for noticing!

File: CAP/gap/MethodRecord.gi (hunk 2 above)
Comment (zickgraf) [reply CAP/gap/MethodRecord.gi]: See above

**Pull request merged**